Sources/App/Core/Search.swift

```diff
@@ -82,6 +84,7 @@ enum Search {
         var lastActivityAt: Date?
         var summary: String?
         var keywords: [String]?
+        var productType: ProductType?
```

Comment:
That was `String?` but that only actually didn't throw on decoding, because previously there was a column name mismatch and the field was always read as `nil` - which isn't actually a problem, because we don't need it's value in the search results display.

The mismatch was that prior to f81902e8e8498cf7eb5e85fe45e792d0808b89ba, the search field we were decoding was `product_type` (see `CodingKeys` below) whereas the `search` view's field at that point was `type`. In f81902e8e8498cf7eb5e85fe45e792d0808b89ba we brought the search view's column name in line with what `DBRecord` was decoding - but then the actual issue arose that the JSONB column can't be decoded into `String?`.

It took me a moment to figure out why this worked originally but then cause a decoding issue! =)